addons/account/models/account_move.py

```diff
@@ -689,7 +689,7 @@ def _compute_name(self):
                     if move._get_last_sequence(lock=False):
                         # The name does not match the date and the move is not the first in the period:
                         # Reset to draft
-                        move.name = '/'
+                        move.name = False if move.quick_edit_mode else '/'
```

Comment:
Why don't you always set it to `False` to avoid double logic?

Reply:
Makes sense